scipy/ndimage/fourier.py

```diff
@@ -111,6 +111,24 @@ def fourier_gaussian(input, sigma, n=-1, axis=-1, output=None):
         The filtered input. If `output` is given as a parameter, None is
         returned.
 
+    Examples
+    --------
+    >>> from scipy import ndimage, misc
+    >>> import numpy.fft
+    >>> import matplotlib.pyplot as plt
+    >>> fig = plt.figure()
+    >>> plt.gray()  # show the filtered result in grayscale
+    >>> ax1 = fig.add_subplot(131)  # left side
+    >>> ax2 = fig.add_subplot(132)  # middle
+    >>> ax3 = fig.add_subplot(133)  # right side
```

Comment:
Very light warning: I don't know what the minimum version of matplotlib is required but if I remember correctly this is a rather new-ish syntax.